### PR TITLE
Update changes for LG webOS Smart TV config flow

### PR DESCRIPTION
**Pull request by thecode**
## Proposed change

- Update instructions for using config flow instead of YAML setup
- Remove broken link to external site
- Update turn on action description and example
- Update description for selecting sources

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/64117
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/webostv.markdown [hunk 1]
```diff
@@ -24,123 +24,43 @@ There is currently support for the following device types within Home Assistant:
...
+        mac: AA-BB-CC-DD-EE-FF
 ```
 
 Any other [actions](/docs/automation/action/) to power on the device can be configured.
 
 ## Sources
 
-To obtain complete list of available sources currently configured on the TV, once the webOS TV is configured and linked, while it's powered on head to the **Developer Tools** > **States**, find your `media_player.<name>` and use the sources listed in `source_list:` remembering to split them per line into your `sources:` configuration. If you leave the `sources:` configuration empty, the `media_player` will offer all sources of the TV. If you list a subset of sources in the configuration, only those will be displayed.
-
-Note: it is normal and expected behavior that for a switched-off TV only the Live TV source is offered in the `media_player`, even if you have configured other sources as well. These will show up as soon as the TV is switched on.
```
Comment (davet2001) [inline source/_integrations/webostv.markdown:143]: Is this note no longer needed?

File: source/_integrations/webostv.markdown [hunk 2]
```diff
@@ -24,123 +24,43 @@ There is currently support for the following device types within Home Assistant:
...
-Please note, the above provides a minimal example, however, all options are
-available for each individual TV.
+{% include integrations/config_flow.md %}
 
 ## Turn on action
 
 Home Assistant is able to turn on a LG webOS Smart TV if you specify an action, like HDMI-CEC or WakeOnLan.
 
-Common for webOS 3.0 and higher would be to use WakeOnLan feature. To use this feature your TV should be connected to your network via Ethernet rather than Wireless and you should enable *LG Connect Apps* feature in *Network* settings of the TV [instructions](https://www.lg.com/uk/support/product-help/CT00008334-1437131798537-others) (or *Mobile App* in *General* settings for older models) (*may vary by version).
+Common for webOS 3.0 and higher would be to use WakeOnLan feature. To use this feature your TV should be connected to your network via Ethernet rather than Wireless and you should enable *LG Connect Apps* feature in *Network* settings of the TV (or *Mobile App* in *General* settings for older models) (*may vary by version).
```
Comment (davet2001) [inline source/_integrations/webostv.markdown]: Suggested edit: "...enable the LG Connect Apps feature..." (add 'the')

**Review by davet2001 — COMMENTED** [review]
Minor edits suggested

File: source/_integrations/webostv.markdown (hunk 1 above)
Comment (thecode) [reply source/_integrations/webostv.markdown:143]: Yes

Comment (davet2001) [reply source/_integrations/webostv.markdown:143]: Ok

File: source/_integrations/webostv.markdown (hunk 2 above)
Comment (thecode) [reply source/_integrations/webostv.markdown]: Added, thanks

**Review by davet2001 — DISMISSED** [review]
Looks good!

**Review by MartinHjelmare — COMMENTED** [review]
Please set the `ha_config_flow` front matter to true at the top of the page.

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**